### PR TITLE
DEV-119: update prereqs for Intro Algo to be consistent with new requirements

### PR DESCRIPTION
**Pull request by RosaGao**

### CODE REVIEW

**Comment by RosaGao** [comment]
image

image

Now the ui doesn't show the alert when only MFCS is taken.

File: data/updatePrereq.js [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+          "Expression": "EN.600.363[C]^OR^EN.601.633[C]",
+          "IsNegative": "Y"
+      }
+  ];
+
+  let data = await SISCourseV.find({number});
+  let course = data[0];
+  for (let version of course.versions) {
+    version.preReq = newPrereq;
+  }
```
Comment (mpark63) [inline data/updatePrereq.js:40]: awesome! we could've selectively updated prereqs for the versions after fall 2021 but this should be fine

File: data/updatePrereq.js [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+}
+
+checkUpdated();
+
+
+
+/** Intro Algo
+ * {
+    "data": {
+        "_id": "635df458662f1ebee559601f",
```
Comment (mpark63) [inline data/updatePrereq.js:65]: looks good, did you run this in both production and development? if not, please do so :)

**Review by mpark63 — APPROVED** [review]

**Comment by mpark63** [comment]
@RosaGao please run this in production

**Comment by mpark63** [comment]
and merge

**Comment by mpark63** [comment]
@RosaGao ready to merge!

**Pull request merged**